generate.go

```diff
@@ -26,6 +26,9 @@ type GenerateOptions struct {
 	// Denotes the number of tokens to predict per generation.
 	MaxTokens uint `json:"max_tokens"`
 
+	// optional - The ID of a custom playground preset.
+	Preset string `json:"preset,omitempty"`
+
```

Comment:
Does it mater if `Prompt` isn't `omitempty`? I image the preset will take precedent over all other params

Reply:
Oh good catch, thanks. It does matter, the params made inside of the request will override the params of the preset.